lib/hpcsocket.js

```diff
@@ -434,8 +441,8 @@ ClusterStatus.prototype.initializeServer = function (callback) {
 
   var self = this;
 
-  var server_socket = ioclient.connect(setup.cluster_ip, {'connect timeout': 1000, 
-                                                          'force new connection': true, 
+  var server_socket = ioclient.connect(default_setup_url, {'connect timeout': 1000,
```

Comment:
This won't work if we are going to check the url's health status when extending the load balance script.